design/features/early access journey

```diff
@@ -0,0 +1,49 @@
+# Editing pages when looking at a preview of a form
+
+## Status
+
+Date created: *2023-11-06*  
+
+Trello card:https://trello.com/c/I8HETm57/1041-design-the-early-access-upgrade-mou-journey-ready-for-user-research 
+Mural working board: https://app.mural.co/t/gaap0347/m/gaap0347/1691659258399/af093136c3f86be05860120cb92d6810e98f3643?sender=u61ceb93b0f861033ca7c7448 
+
+___
+
+## Contents
+
+- [Early access journey](#early-access-journey)
+  - [Status](#status)
+  - [Contents](#contents)
+  - [What](#what)
+  - [Why](#why)
+  - [Hypothesis](#hypothesis)
+
+___
+
+<br>
+
+## What
+With this journey, users can create a trial account and understand if they can upgrade to editor and how to do that. It will also automate a lot of things the adoption team would have to do otherwise, and take us a step closer to self service.
+
+## User needs: 
+As a trial user I need to know what are the criteria to become an editor
+As a trial user I need to know whether someone has agreed to the MOU in my organisation 
+As a GOV.UK Forms team, I need to make sure that organisations have signed an MOU before their users get permissions beyond trial.
+
+<br>
+
+## Why
+
+We currently do not have a process enabling users to request editor access and start publishing forms. 
+
+## Changes
+The changes are in the product. 
+Prototype can be found here: https://forms-prototypes-pr-201.herokuapp.com/product-pages/ 
+
+<br>
+
+___
+
+<br>
+
+[Back to the top](#editing-pages-when-looking-at-a-preview-of-a-form)
```

Comment:
think this is the wrong link?

Reply:
It is the right link, i have added the protoype to the forms protoype.

Reply:
meant this: 

[Back to the top](#editing-pages-when-looking-at-a-preview-of-a-form)

which won't work when you correct the heading.

Reply:
Also I think the idea with this is to record designs here so we can look back at previous versions and why they were the way they were. The prototype is only temporary and can be replaced, deleted, moved. We should only be using the prototype for testing things really.